pages/test_analytics/test_ownership.md

```diff
@@ -105,6 +105,12 @@ You can upload the same TESTOWNERS file to multiple test suites. However, a test
 > 📘
 > You can also create a new pipeline to automatically upload your TESTOWNERS file when changes are detected.
 
+## Viewing test ownership
+
+You can view the current test ownership rules for a test suite in your **Suite settings** page.
```

Comment:
As indicated in our [writing style guidelines](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#ui-elements), I don't see **Suite settings** indicated verbatim within the screenshot you've included.

If **Suite settings** is not indicated anywhere and you were intending to reference the **Test Ownership** tab/page, then I would modify this line to:

```suggestion
You can view the current test ownership rules for a test suite in your **Test Suite** > **Test Ownership** page.
```

The '>' indicating that one might need to click the **Test Ownership** tab to open its page, once initially landing on the **Test Suite** page (which I'm assuming defaults to the **Settings** tab).

Reply:
One would need to navigate from the **Test Suite** page to the **Settings** page which defaults to a **Settings** tab, and finally click on the **Test Ownership** tab.

Would this accurately be represented by the following?

```suggestion
You can view the current test ownership rules for a test suite in your **Test Suite** > **Settings** > **Test Ownership** page
```

Reply:
Thanks for clarifying this @jasmine-q :-)

In that case, it's better to stick with **Test Suite** > **Test Ownership**, since based on the explanation of '>', **Test Suite** > **Settings** > **Test Ownership** implies that you need to click on the **Settings** tab, which from a UX perspective is not the case.